controllers/suggestions/suggest_team_media_controller.py

```diff
@@ -42,11 +42,17 @@ def get(self):
         medias = [media_future.get_result() for media_future in media_futures]
         medias_by_slugname = MediaHelper.group_by_slugname(medias)
 
+        media_key_futures = Media.query(Media.references == team.key, Media.year == None).fetch_async(500, keys_only=True)
+        media_futures = ndb.get_multi_async(media_key_futures.get_result())
+        medias = [media_future.get_result() for media_future in media_futures]
+        social_medias = [media for media in medias if media.media_type_enum == MediaType.INSTAGRAM_PROFILE] # we only allow IG media, so only show IG profile
```

Comment:
Use this DB query instead

https://github.com/the-blue-alliance/the-blue-alliance/blob/master/database/media_query.py#L11

Reply:
Addressed. Let me know if you like/dislike the style, I copied from https://github.com/the-blue-alliance/the-blue-alliance/blob/c59646cdfaef234999017606f17f9aaf83f3136e/renderers/team_renderer.py#L162